packages/frontend-usagers/src/pages/demande-sejour/liste.vue

```diff
@@ -354,8 +360,94 @@ const headers = [
       class: "suivi",
     },
   },
+  {
+    column: "actions",
+    component: (row) => {
+      return {
+        component: DsfrButtonGroup,
+        align: "center",
+        size: "md",
+        inline: true,
+        buttons: [
+          {
+            label: "Dupliquer",
+            iconOnly: true,
+            icon: "ri-file-copy-2-fill",
+            disabled: !["BROUILLON", "TRANSMISE", "EN COURS"].includes(
+              row.statut,
+            ),
+            onClick: (event) => {
+              event.stopPropagation();
+              copyDS(row.demandeSejourId);
+            },
+          },
+          {
+            label: "Supprimer",
+            iconOnly: true,
+            icon: "ri-delete-bin-2-line",
+            onClick: (event) => {
+              event.stopPropagation();
+              deleteDS(row.demandeSejourId);
+            },
+            disabled: !(row.statut === "BROUILLON"),
```

Comment:
```suggestion
            disabled: row.statut !== "BROUILLON",
```

et pareil si on peut faire une enum :D